src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -868,7 +868,11 @@ public final class Address implements ApiMessage {
       fieldMap.put("status", Collections.singletonList(String.valueOf(status)));
     }
     if (fieldNames.contains("users") && users != null) {
-      fieldMap.put("users", Collections.singletonList(String.valueOf(users)));
+      List<String> stringList = new LinkedList<>();
+      for (String item : users) {
+        stringList.add(item.toString());
+      }
+      fieldMap.put("users", stringList);
```

Comment:
I think it would be better to use an `ImmutableList` instead of `LinkedList`.

Reply:
Done.